src/apps/reservation-list/reservation-material/reservation-material.tsx

```diff
@@ -65,7 +65,9 @@ const ReservationMaterial: FC<ReservationMaterialProps & MaterialProps> = ({
         )}
         <ReservationInfo branches={branches} reservationInfo={reservation} />
       </button>
-      <MaterialDetailsModal modalEntity={reservation} material={material}>
+      <MaterialDetailsModal
+        modalId={reservation.faust || reservation.identifier || ""}
```

Comment:
Could it be done in the same way as [here](https://github.com/danskernesdigitalebibliotek/dpl-react/pull/278/files#diff-3a213a7e4253d895c2972925b93e7c4a0808e596803bcf4d46ce2ba5d5af5ee8L21)?

I am asking with absolutely no overview if it makes sense :) But just reckognized a pattern...

Reply:
This sends the modalid into the linked component